### DISCUSSION TITLE
Overview of artists without IMVDB link

### DISCUSSION THREAD

**Discussion by steve1977** [RandomNinjaAtk/docker-lidarr-extended#146] [Q&A] [answered]
Using the video script. AMD used to provide a file naming all artists that don't have an IMVDB link in MBZ. I looked in all folders, but it doesn't seem to have this now? This was an extremely helpful feature to allow me to add IMVDB links to MBZ for all of my artists.

**Comment by RandomNinjaAtk** [comment]
This is a good idea, I'll look into adding it #soon...

**Comment by RandomNinjaAtk** [comment, marked as answer]
Should now be implemented, see here: https://github.com/RandomNinjaAtk/docker-lidarr-extended/commit/a3bff092804ddd4d5697476845acaac53a4d9ded

Will be logged in this folder: `/config/extended/logs/video/imvdb-link-missing`
